lib/EntityGenerator/tests/JoinedTableDefaultValuesResolver.php

```diff
@@ -11,7 +11,7 @@ class JoinedTableDefaultValuesResolver implements DefaultValuesResolverInterface
 {
     public function getDefaultValuesAmongAllFields(NodeTypeFieldInterface $field): array
     {
-        return array_map('trim', explode(',', $field->getDefaultValues()));
+        return $field->getDefaultValuesAsArray();
```

Comment:
Be careful, your `getDefaultValuesAsArray` method does not `trim` array values. You may still get value like:    
`" Page"`